src/styles/index.scss

```diff
@@ -1,3 +1,68 @@
-body {
+$size-stopwatch: 80vmin;
+$size-border: 1vmin;
+
+.page__body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
   margin: 0;
+  width: 100vw;
+  height: 100vh;
+}
+
+.stopwatch {
+  --minute-duration: 60s;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: $size-stopwatch;
+  width: $size-stopwatch;
+  border: $size-border dotted #000;
+  border-radius: 50%;
+  transform-origin: bottom;
+
+  &--speed-up {
+    --minute-duration: 10s;
+  }
+
+  &__minutes-hand {
+    height: 20vmin;
+    width: 3vmin;
+    background-color: #0700ff;
+    position: absolute;
+    transform-origin: bottom;
+    animation: hand-rotate calc(var(--minute-duration, 60) * 60) steps(60)
+      infinite;
```

Comment:
The animation for the minutes hand should use a duration of `calc(var(--minute-duration, 60) * 60s)` to ensure it takes 60 minutes for a full rotation in the default state. This aligns with the task requirement: 'For minutes hand use steps animation with 60 steps. It should take `60min` for the minutes hand to make a full circle.'